maintenance/kubernetes/aks-1.22.md

```diff
@@ -151,7 +151,16 @@ Related to `prodpublicks8` and `temp-privatek8s` clusters
 
         - [x] Expected downtimes: none
 
-      ### Notes
+      ### BUG NOTE AFTER UPGRADE
```

Comment:
```suggestion
### Issues Met during This Upgrade
```
- Important: no indentation
- nit: Title Capitalization ala. "Chicago style" (this nice tool helps me a lot: https://capitalizemytitle.com/)

Reply:
> Title Capitalization ala. "Chicago style"

I hate it so much 😛

Reply:
Your tool adds an uppercase to "During" ;)

Reply:
> Your tool adds an uppercase to "During" ;) (discussing the real issues ^^)

Nope (not that I've selected the "Chicago" tab :) ): 

<img width="1101" alt="Capture d’écran 2022-07-11 à 14 11 19" src="https://user-images.githubusercontent.com/1522731/178261364-e25da5e3-4d90-4618-bba4-10668b0ef220.png">

Reply:
(but it is absolutely nitpicking, I do not care at all which casing do we use: my goal was to share tools and links)

Reply:
I do like that way and didn't know is name. Thanks for the tips.

Reply:
> Nope (not that I've selected the "Chicago" tab :) ):

Oh! Nevermind then.